apps/src/applab/designElements/elementUtils.js

```diff
@@ -120,4 +120,13 @@ module.exports.isIdAvailable = function(newId, originalId) {
   }
 
   return true;
+};
+
+module.exports.isContentEditable = function (element) {
+  return element && element.tagName.toUpperCase() === 'DIV' && element.contentEditable;
```

Comment:
I just realized that a recent PR of mine makes this a little more complicated https://github.com/code-dot-org/code-dot-org/pull/5491. In particular, we have cases where we have a textarea with contenteditable=false.

Something to note:
When we have <div contenteditable=false>, element.contentEditable resolves to `"false"` which is truthy. element.isContentEditable will resolve to `false`

I wonder if what we actually want here is to call this isTextArea instead of isContentEditable. In that case, we'd currently be returning the right result whether the text area was readonly or not. However, we still might want to make this a little more explicit and say `element.contentEditable !== ""`.